### ISSUE TITLE
Repeated 401 error 

### ISSUE THREAD

**Issue by yasunaz** [reportportal/reportportal#598]
**I have a report portal set up as follows:** 
![image](https://user-images.githubusercontent.com/41339223/55210784-7f6d8c80-51bf-11e9-946e-a552f8fd8e1b.png)

**When I run my automated test cases, as you can see from the image below, report portal hangs. ( only running two test cases) :** 

![image](https://user-images.githubusercontent.com/41339223/55210941-31a55400-51c0-11e9-9df8-89a5109f9bcc.png)

**Below is the error that is repeating all the way until my automated test are done:** 

com.epam.reportportal.exception.ReportPortalException: Report Portal returned error
Status code: 401
Status message: Unauthorized
Error Message: 
Error Type: null

	at com.epam.reportportal.service.ReportPortalErrorHandler.handleError(ReportPortalErrorHandler.java:83)
	at com.epam.reportportal.service.ReportPortalErrorHandler.handle(ReportPortalErrorHandler.java:59)
	at com.epam.reportportal.restendpoint.http.HttpClientRestEndpoint$1.subscribe(HttpClientRestEndpoint.java:517)
	at io.reactivex.internal.operators.maybe.MaybeCreate.subscribeActual(MaybeCreate.java:45)
	at io.reactivex.Maybe.subscribe(Maybe.java:4096)
	at io.reactivex.internal.operators.maybe.MaybeCache.subscribeActual(MaybeCache.java:77)
	at io.reactivex.Maybe.subscribe(Maybe.java:4096)
	at io.reactivex.internal.operators.maybe.MaybeSubscribeOn$SubscribeTask.run(MaybeSubscribeOn.java:54)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
00:21:46.054 ERROR c.e.r.u.SubscriptionUtils - rp-io-14 - [39] ReportPortal Finish test item execution error
com.epam.reportportal.exception.ReportPortalException: Report Portal returned error
Status code: 401
Status message: Unauthorized
Error Message: 
Error Type: null

	at com.epam.reportportal.service.ReportPortalErrorHandler.handleError(ReportPortalErrorHandler.java:83)
	at com.epam.reportportal.service.ReportPortalErrorHandler.handle(ReportPortalErrorHandler.java:59)
	at com.epam.reportportal.restendpoint.http.HttpClientRestEndpoint$1.subscribe(HttpClientRestEndpoint.java:517)
	at io.reactivex.internal.operators.maybe.MaybeCreate.subscribeActual(MaybeCreate.java:45)
	at io.reactivex.Maybe.subscribe(Maybe.java:4096)
	at io.reactivex.internal.operators.maybe.MaybeCache.subscribeActual(MaybeCache.java:77)
	at io.reactivex.Maybe.subscribe(Maybe.java:4096)
	at io.reactivex.internal.operators.maybe.MaybeSubscribeOn$SubscribeTask.run(MaybeSubscribeOn.java:54)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
	at io.reactivex.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:26)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
00:21:46.166 DEBUG c.e.r.s.Launch - rp-io-13 - ReportPortal item with ID '5c9d9d59b3cb1f0001351157' has been created
00:21:46.167 DEBUG c.e.r.u.SubscriptionUtils - rp-io-13 - Start test item successfully completed

**Comment by gkalnytskyi** [comment]
I have similar ReportPortal setup and facing the similar issue. I'm using ReportPortal pluging for Specflow and followed [Specflow setup instructions](https://github.com/reportportal/agent-net-specflow). But my client cannot authorize.

Here is a first line of the exception:

    System.AggregateException: One or more errors occurred. ---> System.Exception: Cannot finish launch due inner items failed to finish. ---> System.AggregateException: One or more errors occurred. ---> System.Exception: Cannot finish test item due starting item failed. ---> System.AggregateException: One or more errors occurred. ---> System.Exception: Cannot start a test item due parent failed to start. ---> System.AggregateException: One or more errors occurred. ---> System.Net.Http.HttpRequestException: Unexpected response status code. POST http://dadplx206.dev.adp.asx.com.au:8080/api/v1/bbsw_aonia/launch
Response Body: {"error":"invalid_token","error_description":"7853c7a9-7f27-43ea-835a-cab01355fd17"} ---> System.Net.Http.HttpRequestException: Response status code does not indicate success: 401 (Unauthorized).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at ReportPortal.Client.Extentions.HttpResponseMessageExtension.VerifySuccessStatusCode(HttpResponseMessage httpResponseMessage) in C:\projects\client-net\ReportPortal.Client\Extentions\HttpResponseMessageExtension.cs:line 10
   --- End of inner exception stack trace ---
   at ReportPortal.Client.Extentions.HttpResponseMessageExtension.VerifySuccessStatusCode(HttpResponseMessage httpResponseMessage) in C:\projects\client-net\ReportPortal.Client\Extentions\HttpResponseMessageExtension.cs:line 19
   at ReportPortal.Client.Service.<StartLaunchAsync>d__14.MoveNext() in C:\projects\client-net\ReportPortal.Client\Service.Launch.cs:line 58

Can you please advice how I can check what authorization token I should use for a specific project?

**Comment by DzmitryHumianiuk** [comment]
@gkalnytskyi .net related question for @nvborisenko 

@yasunaz java related might be for @filland and @pbortnik 

but @yasunaz do you add cert information within `reportportal.properties` file?

**Comment by nvborisenko** [comment]
@gkalnytskyi incorrect token. This token can be obtained from user profile page (Web UI).

**Comment by gkalnytskyi** [comment]
@nvborisenko , thank you for suggestion, I have resolved this issue.

@DzmitryHumianiuk 
But I see another issue here. If I want to have a default reporter for my CI process, whose UUID is saved in the VCS. I need to login as that user to find out his UUID, even with ReportPortal admin permissions.

**Comment by DzmitryHumianiuk** [comment]
@gkalnytskyi yes.
it's the way how it build for now.

i'll keep it in mind for UX updates.
I guess, we will introduce auto-generated user for usage in CI tools

**Comment by DzmitryHumianiuk** [comment]
@gkalnytskyi plz submit it as a separate feature request.

this issue can be closed

**Comment by yasunaz** [comment]
@DzmitryHumianiuk @gkalnytskyi @nvborisenko @lexecon 
I have resolved this issue, as it turns out, UUID was constantly changing every-time I log in.  Also, report portal would log me out when I clicked tabs, button etc..
I have tried to re-create report portal in my server with almost all version of docker-compose file availble in github page, all of it gave me the same issue except following configuration:
```
mongo: 3.4
consul:1.0.6
service-authorization:4.2.0
fabio:1.5.8-go1.10
service-index:4.2.0
service-api:4.3.4
service-ui:4.3.6
```
recreating report portal with this set-up eliminated all of the problem I was facing. So far this is the most stable report portal experience for me.  Thanks for your responses, you guys rock!
